pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -39,6 +39,11 @@ const ServiceAnnotationLoadBalancerInternal = "service.beta.kubernetes.io/azure-
 // to specify what subnet it is exposed on
 const ServiceAnnotationLoadBalancerInternalSubnet = "service.beta.kubernetes.io/azure-load-balancer-internal-subnet"
 
+// ServiceAnnotationLoadBalancerPublicIPAddressResourceGroup is the annotation used on the service
+// to specify the resource group of the public IP address,
+// which can be used to specify public IP address that's not in the same resource group as the cluster
+const ServiceAnnotationLoadBalancerPublicIPAddressResourceGroup = "service.beta.kubernetes.io/azure-load-balancer-pip-resource-group"
```

Comment:
Let's remove the 'pip' part in case we want to re-use this for other load balancer objects...

Reply:
Removed. Are we assuming other load balancer objects will share the same resource group?